src/annotator/components/test/NotebookModal-test.js

```diff
@@ -0,0 +1,135 @@
+import NotebookModal from '../NotebookModal';
+import { EventBus } from '../../util/emitter';
+import { mount } from 'enzyme';
+import { act } from 'preact/test-utils';
+
+describe('NotebookModal', () => {
+  let container;
+  let components;
+  let eventBus;
+
+  const createComponent = config => {
+    const component = mount(
+      <NotebookModal
+        eventBus={eventBus}
+        config={{ notebookAppUrl: '/notebook', ...config }}
+      />,
+      {
+        attachTo: container,
+      }
+    );
+    components.push(component);
+    return component;
+  };
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    components = [];
+    document.body.appendChild(container);
+    eventBus = new EventBus();
+  });
+
+  afterEach(() => {
+    components.forEach(component => component.unmount());
+    container.remove();
+  });
+
+  it('hides modal on first render', () => {
+    const wrapper = createComponent();
+    const outer = wrapper.find('.Notebook__outer');
+
+    assert.isTrue(outer.hasClass('is-hidden'));
+    assert.equal(getComputedStyle(wrapper.getDOMNode()).display, 'none');
```

Comment:
What is this assert for? Is this testing the effect of the `is-hidden` class? If so we wouldn't usually do this in a unit test given how simple the `is-hidden` class is. If there was a good reason to do that, I would suggest to at least extract the duplicated sequence of asserts into a helper.

Reply:
I will remove this line.

I prefer to test the UI more than the presence of a class. I understand this is a very simple case, but if someone would accidentally remove the CSS the test would still succeed but without the intended result.